packages/calcite-components/src/components/sheet/sheet.scss

```diff
@@ -257,13 +272,15 @@
 .resize-handle:active,
 .resize-handle:hover {
   .resize-handle-bar {
-    @apply text-color-1 bg-foreground-3;
+    color: var(--calcite-sheet-resize-text-color, var(--calcite-color-text-1));
+    background-color: var(--calcite-sheet-resize-background-color, var(--calcite-color-foreground-3));
   }
 }
 
 .resize-handle-bar {
-  @apply flex bg-background justify-center items-center pointer-events-none;
-  color: var(--calcite-color-border-input);
+  @apply flex justify-center items-center pointer-events-none;
+  color: var(--calcite-sheet-resize-text-color, var(--calcite-color-border-input));
```

Comment:
Naming q here - seems strange to name this text when its an icon. cc @matgalla @ashetland

Reply:
Changed to `-icon`